scripts/synapse_port_db

```diff
@@ -128,10 +128,20 @@ APPEND_ONLY_TABLES = [
 
 
 IGNORED_TABLES = {
+    # We don't port these tables, as they're a faff and we can regenerate
+    # them anyway.
     "user_directory",
     "user_directory_search",
-    "users_who_share_rooms",
-    "users_in_pubic_room",
+    "user_directory_search_content",
+    "user_directory_search_docsize",
+    "user_directory_search_segdir",
+    "user_directory_search_segments",
+    "user_directory_search_stat",
+    "user_directory_search_pos",
+    "users_who_share_private_rooms",
+    "users_in_public_room",
```

Comment:
These two tables were renamed at some point, I believe.

Reply:
Looks like they were. They're regenerated because somewhere later in the port code we explicitly reset the entry in the `user_directory_stream_pos` table